### PR TITLE
Feature/e UI 8455

### PR DESCRIPTION
**Pull request by MateiHMCTS**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-8455

### Change description ###
Display multiple responses and follow up

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by johnbenjamin-hmcts — APPROVED** [review]

**Review by DavidJayakumar — DISMISSED** [review]
Looks good

**Review by DavidJayakumar — APPROVED** [review]
Looks good

**Review by johnbenjamin-hmcts — APPROVED** [review]

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/query-management/components/query-details/query-details.component.html [hunk 1]
```diff
@@ -1,7 +1,71 @@
...
+            </tr>
+          </tbody>
+        </table>
+      </ng-template>
+    </ng-container>
+  </ng-container>
+
+  <ng-template #queryIsInReview>
+    <div>
+      <p class="govuk-!-font-weight-bold">Your query is under review</p>
```
Comment (DavidJayakumar) [inline projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/query-management/components/query-details/query-details.component.html:67]: Doesn't have translation

**Review by DavidJayakumar — CHANGES_REQUESTED** [review]
@MateiHMCTS Just one comment

**Review by johnbenjamin-hmcts — DISMISSED** [review]

**Review by DavidJayakumar — DISMISSED** [review]
@MateiHMCTS Looks good

**Review by DavidJayakumar — APPROVED** [review]
LGTM

**Review by johnbenjamin-hmcts — APPROVED** [review]

**Pull request merged**